contributions/catalog/requirements.txt

```
@@ -1,4 +1,6 @@
 Flask==1.1.1
```

```suggestion
Flask==1.1.1
werkzeug==2.0.2
```
add additional requirement

---

Hi, @bingzhang, it looks like this suggestion was resolved, but the `werkzeug` change has not been added to the catalog/requirements.txt. Please fix this. I'm unresolving this comment. Thanks.

---

Thanks, Bing. I will mark this resolved.